robot/Cumulus/resources/locators_48.py

```diff
@@ -232,6 +235,12 @@
 
     'object_manager':{
         'button':'//input[@title="{}"]',
+        'Lookup_option':'//td/input[@id = "dtypeY"]',
+        'select_related':'//select[@id = "DomainEnumOrId"]',
+        'select_related_option':'//select[@id = "DomainEnumOrId"]/option[@value="{}"]',
+        'input_field_label': '//input[@id="MasterLabel"]',
+        'global_search':'//input[@id="globalQuickfind"]',
```

Comment:
can line 238, 241 and 242 use same locator //input[@id='{}'] and pass id in the keyword def

Reply:
This gets slightly complicated and not so readable because each of them comes with a unique id like dtypeY, dtypeX, so from the code you will have to format option.format(dtypeX) etc.

Reply:
yes by doing that u r making ur locators unique for that keyword. If u hardcode ids in the locator's file, we will not be making our locators reusable but we will end up with lots of locators in our locator file. Levels dictionary already contains a locator for input with id, please see if you can repurpose it and if yes move it to common locators and make adjustments for existing as well as new test

Reply:
Updated to a single locator